<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>235</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alteração de Dados</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/alterar.png</normaloff>icons/alterar.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Alteração de dados</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ID do Aluno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="id_aluno"/>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nome do Aluno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nome_aluno"/>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Quantidade de Dias</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="quantidade_dias"/>
          </item>
          <item>
           <widget class="QCheckBox" name="check_status">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_alterar">
            <property name="text">
             <string>Alterar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
